test/rubygems/test_gem_commands_setup_command.rb

```diff
@@ -27,6 +27,28 @@ def setup
     open 'bundler/exe/bundle',        'w' do |io| io.puts '# bundle'       end
     open 'bundler/lib/bundler.rb',    'w' do |io| io.puts '# bundler.rb'   end
     open 'bundler/lib/bundler/b.rb',  'w' do |io| io.puts '# b.rb'         end
+
+    FileUtils.mkdir_p 'default/gems'
+
+    gemspec = Gem::Specification.new
+    gemspec.name = "bundler"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation of special symbols.